indra/assemblers/html/template.html

```diff
@@ -208,13 +236,116 @@ <h4 class="align-middle">{{ stmt_info['english'] }}
               </tr>
             </thead>
             <tbody>
-              {% for ev in stmt_info['evidence'] %}
-              <tr>
-                  <td>&#9998;
-                  <td>{{ ev['source_api'] }}</td>
-                  <td>"{{ ev['text'] }}"</td>
-                  <td><a class="pmid_link" href='https://www.ncbi.nlm.nih.gov/pubmed/{{ ev['pmid'] }}'>{{ ev['pmid'] }}</a></td>
-              </tr>
+                {% for ev in stmt_info['evidence'] %}
+                <tr class="parent">
+                    <td class="curation_toggle">&#9998;</td>
+                    <td>{{ ev['source_api'] }}</td>
+                    <td>"{{ ev['text'] }}"</td>
+                    <td>
+                        <a class="pmid_link"
+                href='https://www.ncbi.nlm.nih.gov/pubmed/{{ ev['pmid'] }}'>
+                        {{ ev['pmid'] }}
+                        </a>
+                    </td>
+                </tr>
+                <tr class="cchild">
+                    <td colspan="4" style="padding: 0px">
+                    <div class="dropdown" 
+                         style="display:inline-block; vertical-align: middle;">
+                        <select>
+                            <!-- "option" before choosing an option -->
+                            <option value="" selected disabled hidden>
+                                Select error type
+                            </option>
+                            <!-- ERROR DROP DOWN OPTIONS -->
+                            <option value="evidence">evidence</option>
+                            <option value="polarity">polarity</option>
+                            <option value="grounding">grounding</option>
+                            <option value="other">other...</option>
+                        </select>
+                    </div>
+                    <div class="form" 
+                         style="display:inline-block; 
+                                vertical-align: middle; 
+                                top: 0px">
+                        <form name="user_feedback_form">
+                            <input type="text" 
+                                   name="user_feedback" 
+                                   placeholder="Describe the error with 240 characters" 
+                                   value="" 
+                                   style="width: 360px;">
+                        </form>
+                    </div>
+                    <div id="n" 
+                         class="curation_button"
+                         style="display:inline-block; 
+                                vertical-align: middle;">
+                        <button
+                            id="n"
+                            class="btn btn-default btn-submit pull-right"
+                            style="padding: 2px 6px">submit
+                        </button>
+                        <script type="text/javascript">
```

Comment:
Is it possible to implement this without the script being duplicated across all sentences in the output? I imagine this will significantly increase the size of the returned HTML file for large queries.

Reply:
Yes, it should be possible to move all this to the script section as a function and then call that function.

I have a weak hunch that might break the button uniqueness, but if it does it's straightforward to solve once we have button IDs in place.